kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -255,3 +245,18 @@ def cdgmm(A, B, inplace=False):
         C[..., 1].view(-1, C.size(-2)*C.size(-3))[:] = A_r * B_i + A_i * B_r
 
         return C if not inplace else A.copy_(C)
+
+def new(x, O, M, N):
```

Comment:
this function still seems super ad hoc. can you write a docstring about what it is for?

Reply:
I removed it. It's simpler now